src/integrationTest/java/com/mongodb/spark/sql/connector/write/MongoSparkConnectorWriteTest.java

```diff
@@ -111,6 +115,83 @@ void testIgnoreNullValues() {
         dataWithoutNulls.stream().map(BsonDocument::parse).collect(Collectors.toList()));
   }
 
+  @Test
+  void testIgnoreDuplicates() {
```

Comment:
Can we add a test for mixed error scenarios (duplicate + non-duplicate errors in same batch)?

Reply:
We test duplicate errors and successful writes. Not sure how I'd trigger a non duplicate error using inserts 🤷